Makefile

```diff
@@ -60,16 +64,22 @@ pre-docker-test:
 	mkdir -p ~/tmp/apiplatform/api-gateway-cachemanager/target/test-logs
 	ln -s ~/tmp/apiplatform/api-gateway-cachemanager/target/test-logs ./target/test-logs
 
+.PHONY: get-redis-docker-ip
+get-redis-docker-ip:
+	$(eval $@_IP := $(shell docker run --entrypoint=ifconfig alpine eth0 | grep "inet addr" | cut -d: -f2 | awk '{ print $$1}'))
+	@echo "Assuming the next IP for the docker container is:" $($@_IP)
+	sed -i '' 's/127\.0\.0\.1\:6379/$($@_IP)\:6379/g' ~/tmp/apiplatform/api-gateway-cachemanager/test/perl/api-gateway/cache/status/remoteCacheStatus.t
+
 post-docker-test:
 	echo "    post-docker-test"
-	# cp -r ~/tmp/apiplatform/api-gateway-cachemanager/target/ ./target
-	# rm -rf  ~/tmp/apiplatform/api-gateway-cachemanager
+	cp -r ~/tmp/apiplatform/api-gateway-cachemanager/target/ ./target
+	rm -rf  ~/tmp/apiplatform/api-gateway-cachemanager
 
 run-docker-test:
```

Comment:
@ddragosd `make test-docker` is not available anymore, as described in the [documentation](https://github.com/adobe-apiplatform/api-gateway-cachemanager#developer-guide).